spec/mailers/previews/hyku_mailer_preview.rb

```diff
@@ -21,4 +21,25 @@ def summary_email
 
     HykuMailer.new.summary_email(user, messages, account)
   end
+
```

Comment:
to view, create an acme tenant and visit http://acme.pals.test/rails/mailers/hyku_mailer/depositor_email